vignettes/config_yml.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: documentation of options in config.yml
+title: Configuring your application
```

Comment:
(incredibly) NIT: keep titles symmetric with each other?
```suggestion
title: Configuring the application
```

Reply:
tend to agree